config/custom-environment-variables.json

```diff
@@ -10,5 +10,10 @@
     "port": "COUCHDB_PORT",
     "username": "COUCHDB_USER",
     "password": "COUCHDB_PASS"
+  },
+  "gitlabOAuth": {
```

Comment:
In https://github.com/jum-s/data.transformap.co/blob/7f3a6229300775f7e08417f9a1dac7708c15ec44/server/middlewares/strategies/gitlab.js#L8-L12 this is merely called `gitlab`. It also mentions further parameters like

* `authorizationURL` and
* `tokenURL`